nx/lib/nx/lin_alg.ex

```diff
@@ -1022,20 +1022,35 @@ defmodule Nx.LinAlg do
         ]
       >
 
+      iex> a = Nx.tensor([[1, Complex.new(0, 2), 2], [Complex.new(0, -2), -3, Complex.new(0, 2)], [2, Complex.new(0, -2), 1]])
```

Comment:
we should probably keep this as a test instead of a doctest due to readability on the eigenvec matrix